index.html

```diff
@@ -0,0 +1,251 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Project 1</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <!-- // nav bar start  -->
+    <nav>
+      <div class="icon">
+        <div class="Micon">
+          <img src="/Icon/Menu.png" width="24" height="24" />
+        </div>
+        <div class="Yicon"><img src="/Icon/Youtube icon.png" /></div>
+      </div>
+      <div class="search" style="display: flex; flex-direction:row">
+        <div class="searchbar">
+          <input
+            type="search"
+            name="search"
+            id="searchbar"
+            placeholder="Search"
+          />
+          <input type="button" value="" id="Searchbutton" />  
+        </div>
+      </div>
+      <div class="mic">
+        <img src="./Icon/icons8-microphone-24.png" alt="" />
+      </div>
+      <div class="end">
+        <img src="./Icon//icons8-live-video-on.gif" alt="" id="video" />
+        <img
+          src="./Icon//notifications_FILL0_wght400_GRAD0_opsz48.png"
+          alt=""
+          id="noti"
+        />
+        <div class="circle"></div>
+      </div>
```

Comment:
Kindly wrap your toolbar in a <header> tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.